### DISCUSSION TITLE
Change code suggestion text color

### DISCUSSION THREAD

**Discussion by QarthO** [zed-industries/zed#20251] [Feature Requests]
Been using zed with SuperMaven as my ai code suggestion.

Is there anyway to change the text color of the suggested code? In a lot of the files its very hard to tell what code is being suggested vs what code is already written.

Can you tell where my cursor is on this line? This is an accessibility nightmare.
<img width="311" alt="image" src="https://github.com/user-attachments/assets/bbf94369-8dd3-409a-bcbc-e23c7257a1bb">

For reference, its very clear to tell what code is present, vs what is ai suggested in other editors
<img width="241" alt="image" src="https://github.com/user-attachments/assets/fa0c5ab3-1b1f-415c-85a7-061e8e426795">

**Comment by bajrangCoder** [comment]
> Is there anyway to change the text color of the suggested code? In a lot of the files its very hard to tell what code is being suggested vs what code is already written.

Try changing these theme colors :

```jsonc
"ghost_element.background": null,
"ghost_element.hover": null,
"ghost_element.active": null,
"ghost_element.selected": null,
"ghost_element.disabled": null,
// Or
"text.muted": "null",
```

> I haven't tried it.

You can provide these in your zed settings, under experimental theme override.

Btw I guess this is theme problem, as zed uses the colors provided by the theme.

**Comment by QarthO** [comment]
> Btw I guess this is theme problem, as zed uses the colors provided by the theme.

Appreciate the message! **This was theme related**, I tried all of the suggested changes however none of them changed the text color for ai suggestions, (also ``null`` didn't seem like a supported value). 

Instead,  just switched to a different theme, and the problem has been solved. Thank you!
